libs/agno/agno/workflow/workflow.py

```diff
@@ -972,8 +972,13 @@ def _handle_event(
         event: "WorkflowRunOutputEvent",
         workflow_run_response: WorkflowRunOutput,
         websocket_handler: Optional[WebSocketHandler] = None,
-    ) -> "WorkflowRunOutputEvent":
+    ) -> Optional["WorkflowRunOutputEvent"]:
         """Handle workflow events for storage - similar to Team._handle_event"""
+        from agno.run.agent import RunOutput
+        from agno.run.team import TeamRunOutput
+
+        if isinstance(event, RunOutput) or isinstance(event, TeamRunOutput):
```

Comment:
i think will be cleaner if we do `if isinstance(event, (RunOutput, TeamRunOutput)):`